MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -329,34 +311,21 @@ def main():
     # make a CCDB accessor object
     ccdbreader = CCDBAccessor(args.ccdb_url)
     # fetch the EOR/SOR
-    rct_sor_eor = retrieve_sor_eor(ccdbreader, args.run_number) # <-- from RCT/Info
-    GLOparams = retrieve_params_fromGRPECS(ccdbreader, args.run_number, rct=rct_sor_eor)
+    run_duration = ccdbreader.get_run_duration(args.run_number)
+    run_start = run_duration.first
+    run_end = run_duration.second
+
+    GLOparams = retrieve_params_fromGRPECS(ccdbreader, args.run_number, run_start=run_start, run_end=run_end)
```

Comment:
Is it needed to pass run_start and run_end here?

Reply:
Can we simply not avoid to retrieve it from GRPECS now?

Reply:
We get the orbit info from that. I asked @shahor02 specifically via email if we should stick to that method and as far as I understand, that should be the way. So we need to process `GRPECS` for that reason.

Reply:
Yes, but you can avoid retrieving SOR and EOR, this is what I mean. They will not be consistent with RCT.

Reply:
Thanks! This is what I meant.